src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs

```diff
@@ -28,6 +28,75 @@ public JsonConsoleFormatter(IOptionsMonitor<JsonConsoleFormatterOptions> options
             _optionsReloadToken = options.OnChange(ReloadLoggerOptions);
         }
 
+        private static List<Exception> GetInnerExceptions(Exception exception)
+        {
+            List<Exception> result = new List<Exception>();
+            if (exception.InnerException != null)
+            {
+                result.Add(exception.InnerException);
+            }
+            AggregateException aggregated = exception as AggregateException;
+            if (aggregated != null)
+            {
+                foreach (var inner in aggregated.InnerExceptions)
+                {
+                    if (inner != exception.InnerException)
+                    {
+                        result.Add(inner);
+                    }
+                }
+            }
+            return result;
+        }
+
+        private static void WriteException(Utf8JsonWriter writer, Exception exception, int maxDepth, bool includePropertyName)
+        {
+            if (includePropertyName)
+            {
+                writer.WriteStartObject(nameof(Exception));
+            }
+            else
```

Comment:
perhaps remove the else case and includePropertyName argument, as it is always true.

Reply:
NO: it should not be always true (true on first call, false on every deeper call...) without the Jsonwriter throws exception because called after "StartArray"....

Reply:
Oh right, I see that now.